### DISCUSSION TITLE
Error after updating to Version 1.13.0 for Kibana 7.16.2

### DISCUSSION THREAD

**Discussion by mlnskr** [fbaligand/kibana-enhanced-table#253] [Q&A] [answered]
Hi, 

I just updated the plugin to version 1.13.0 for kibana 7.16.2. After starting Kibana I get the following error. Is there something wrong with the plugin?

`[$injector:modulerr] Failed to instantiate module kibana/enhanced_table_vis due to: [$injector:modulerr] Failed to instantiate module tableVisI18n due to: [$injector:pget] Provider 'i18n' must define $get factory method. https://errors.angularjs.org/1.8.2/$injector/modulerr?p0=kibana%2Fenhanced_table_vis&p1=%5B%24injector%3Amodulerr [....]`

Thank you in advance.

**Comment by fbaligand** [comment, marked as answer]
Hi,

After test, I confirm that I reproduce your issue with Kibana 7.16.2.
But I don't reproduce the issue with Kibana 7.16.3.
That's why I invite you to upgrade your Kibana version to 7.16.3.

**Reply by mlnskr** [reply]
After updating Kibana and the Plugin to Version 7.16.3 everything works  fine. Thanks for your help.
